src/services/helper/entity.js

```diff
@@ -28,6 +28,8 @@ module.exports = class EntityHelper {
 				})
 			}
 			await entitiesData.createEntity(bodyData)
+			const key = 'mentoring_entity_' + bodyData.type
```

Comment:
@ankitpws can we change this to _id_entity_type ? or any better way.

Reply:
@rakeshSgr we are not receiving id in API call so we can make id that's why i have used mentoring because in users module we have entity service